### DISCUSSION TITLE
pdtm does not find the binaries that are in the $PATH

### DISCUSSION THREAD

**Discussion by ulises2k** [projectdiscovery/pdtm#107] [Q&A]
![image](https://user-images.githubusercontent.com/2415609/226933405-981c4c26-e383-454e-9635-de1e45c01d4b.png)
It does not find the binaries that are in the $PATH
![image](https://user-images.githubusercontent.com/2415609/226933642-99381c3d-a49b-4410-a229-7623596c0c6b.png)

**Comment by ehsandeep** [comment]
@ulises2k this is intentional, I will make a note of it in the readme, but to avoid confusion and overwriting arbitrary system installed binary, pdtm's scope is limited to tools installed using pdtm under pdtm binary location.

**Comment by ulises2k** [comment]
In previous versions of pdtm it looked for the binaries that were in the $PATH

**Reply by ehsandeep** [reply]
Right, this is something got updated in latest release (https://github.com/projectdiscovery/pdtm/pull/64)
